Unverified Buyer List

tremainebuchanan opened the pull request:
List of unverified buyers now show to admin user. Admin user can now update status of a buyer.

tremainebuchanan committed:
Minor mods

tremainebuchanan committed:
Merge branch 'develop' into unverified-buyer-list-impl

tremainebuchanan committed:
Fixed issue with cancel button

tremainebuchanan committed:
Admin now able to view unverified buyers.
List of unverified buyers now show to
admin user. Admin user can now update
status of a buyer.

tremainebuchanan commented on public/javascripts/controllers/admin-ctrl.js:
```diff
@@ -1,8 +1,9 @@
 angular.module('jasmic.controllers')
-    .controller('AdministratorCtrl', ['$scope','$mdDialog', '$mdToast','UsersFactory','CropsFactory','SuppliersFactory',
+    .controller('AdministratorCtrl', ['$scope','$mdDialog', '$mdToast','$window','UsersFactory','CropsFactory','SuppliersFactory',
     'UserFactory','CropFactory','SupplierFactory', 'EmailFactory', 'UserProfileFactory', 'ParishesFactory',
```
@matjames007 I encountered an issue after updating a buyer. The screen would turn grey but the dialog box was not seen. In testing the branch, you can remove the $window.scroll line to see if you get encounter this behaviour.

matjames007 replied:
I haven't seen the behaviour described.  I also haven't seen the `$window.scroll` line u mentioned.  Could you point me to the file and line number on this branch?

tremainebuchanan committed:
Removed test function.

tremainebuchanan commented on public/javascripts/controllers/admin-ctrl.js:
```diff
@@ -236,6 +264,7 @@ angular.module('jasmic.controllers')
        */
       function updateSupplier(){
          SupplierFactory.update({id:$scope.supplier._id}, $scope.supplier, function(success){
+              $window.scrollTo(0,0);
```
@matjames007  here's the reference to the line.

matjames007 replied:
oh when u are updating the supplier! not the buyer...lol ok

tremainebuchanan replied:
@matjames007 actually both! lol..

matjames007 committed:
This should solve the problem encountered when the dialog doesnt centre properly.
Now the issue is that the parent is usually the body element and it will always
try to centre relative to the top of the page.  So the best way to centre is
based on the object being manipulated.

I wrote code to pull the item being manipulated and set as dialog parent.
Achieved by using an overloaded function.

Matthew Budram

matjames007 committed:
Merge branch 'develop' into unverified-buyer-list-impl

Conflicts:
	public/javascripts/controllers/admin-ctrl.js

tremainebuchanan committed:
Merge branch 'develop' into unverified-buyer-list-impl

tremainebuchanan committed:
Merge branch 'unverified-buyer-list-impl' of https://github.com/slashroots/jas-mis-app into unverified-buyer-list-impl

tremainebuchanan committed:
Merge branch 'develop' into unverified-buyer-list-impl

Conflicts:
	public/javascripts/controllers/admin-ctrl.js
